plugins/in_winevtlog/pack.c

```diff
@@ -377,10 +380,14 @@ static void pack_string_inserts(struct winevtlog_config *ctx, PEVT_VARIANT value
             }
             break;
         default:
-            msgpack_pack_str(ctx, 1);
-            msgpack_pack_str_body(ctx, "?", 1);
+            flb_log_event_encoder_append_body_cstring(ctx->log_encoder, "?");
         }
     }
+
+    if (ret == FLB_EVENT_ENCODER_SUCCESS) {
+        ret = flb_log_event_encoder_body_commit_array(ctx->log_encoder);
```

Comment:
Should this `ret` value be checked as well?

Reply:
Normally yes, for new code **definitely**, for older code like this it might be acceptable to skip it since it's not worse than the original behavior.

Reply:
Oh, sorry for lacking of the descriptions to this change.
This should be mainly for restoring behavior of the previous code before metadata migration.